tdvt/tdvt/test_results.py

```diff
@@ -98,14 +101,15 @@ def table_to_json(self):
                 for v in t.findall('value'):
                     tuple_list.append(v.text)
 
-        return {'tuples' : tuple_list}
+        return {'tuples': tuple_list}
 
     def __json__(self):
-        return {'tested_sql' : self.tested_sql, 'tested_tuples' : self.tested_tuples, 'id' : self.id, 'name' : self.name, 'sql' : self.get_sql_text(), 'table' : self.table_to_json()}
```

Comment:
I know it's a hassle, but could you format the `return` statement here and in tdvtconfig.py:74 so they won't raise a flake8 error?

Reply:
Formatted to complying flack8 config.